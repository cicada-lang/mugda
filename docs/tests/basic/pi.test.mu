(Pi ([A Type]) (-> A A))

(Pi ([A Type] [B Type]) (-> A B Type))
