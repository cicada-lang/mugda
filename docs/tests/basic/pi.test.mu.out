(Pi ([A Type] [_ A]) A)
(Pi ([A Type] [B Type] [_ A] [_ B]) Type)
